### DISCUSSION TITLE
Allow use of Proton SMTP servers as MX records for SimpleLogin custom domains

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1237] [Feature Requests]
Now that you’re integrated/integrating with Proton Mail it would be nice if we could use their SMTP servers as MX records for custom domains

Given some websites discriminate against using email alias services by blacklisting the domain name, others take it further by looking up the MX records of the domain. If they see SimpleLogin MX records they deny the use of the custom domain

By allowing the use of Proton Mail’s MX records for custom domains this would further obfuscate the service behind the domain, which is a win for privacy imo

Unless the service blocks Proton they would be none the wiser that the user is using SimpleLogin as the MX records would point to Proton instead.

Thoughts?

**Comment by JasonMoriarty** [comment]
I wanna agree with you. but I'm worried about they may just reject Proton MX records as well after this change.

**Comment by ghost** [comment]
If they block Proton altogether I think there'll be bigger problems than SimpleLogin

This is why I only mentioned custom domains, as opposed to any of the SimpleLogin owned ones

To be perfectly honest if a website/service went to that extreme then I wouldn't sign up as they clearly don't value privacy

**Reply by mjgardner** [reply]
> To be perfectly honest if a website/service went to that extreme then I wouldn't sign up as they clearly don't value privacy

They already don't value privacy if they're already blocking SimpleLogin

**Comment by hcurtiss** [comment]
For what it's worth, Fastmail basically works this way.  Their masked email addresses are handled behind their MX servers (ns1.messagingengine.com).  There's no way for a service to know it's a masked email.  This has me a little bit nervous about transitioning entirely to Simplelogin's service, even though it's MUCH more convenient to use.

**Reply by JasonMoriarty** [reply]
For me, it works fine. So far this has barely happened.
